electron/ui/src/views/FlowsheetConfig/FlowsheetConfig.js

```diff
@@ -134,6 +141,9 @@ export default function FlowsheetConfig() {
       setOpenSuccessSaveSnackbar(false);
     };
 
+    const handleErrorSnackbarClose = () => {
```

Comment:
ditto.. can we get rid of the "Snackbar" part of these names (in a separate PR)?

Reply:
I added this one as a pop up that opens when an error occurs when running the model (ie bad data input). Snackbar is just the name of the MUI component that pops up, but I can create a better name for it, as well as improve the error handling a bit